addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -58,10 +58,10 @@ def default_get(self, fields):
     def _compute_medium_id(self):
         super(Mailing, self)._compute_medium_id()
         for mailing in self:
-            if mailing.mailing_type == 'sms' and (not mailing.medium_id or mailing.medium_id == self.env.ref('utm.utm_medium_email')):
+            if mailing.mailing_type == 'sms' and (not mailing.medium_id or mailing.medium_id == self.env['utm.medium']._fetch_or_create_utm_medium('email')):
                 mailing.medium_id = self.env.ref('mass_mailing_sms.utm_medium_sms').id
```

Comment:
Could also add this one through override ? Seems it becomes a master data with SMS

Reply:
As I checked this medium is already protected via a method set ondelete in the `utm.medium` override:
[here](https://github.com/odoo-dev/odoo/blob/e9a71189995f7e36e126aa0e5c8a5e1b450991b8/addons/mass_mailing_sms/models/utm.py#L84)
I don't think it's needed here.

Reply:
But there is no find or create for this one, could be supported I guess ?